### PR TITLE
[sites] fix duplicated `feed:` property

### PR DESCRIPTION
**Pull request by dym-sh**
...forgot to make another branch

also fixed the "undefined" class on older sites w/o `type:`
and unified output for whatever `<type>` might be

### CODE REVIEW

**Comment by kodedninja** [comment]
I like the changes, but as you touched @neauoire's entry also, I would like him to check it too.

File: scripts/portal.js [hunk 1]
```diff
@@ -17,11 +17,13 @@ function Portal (sites) {
 
   function _directory (sites) {
     return `
-    <ul>${sites.reduce((acc, site, id) => { return `${acc}<li class='${site.type}'><span class='counter'>${id})</span><a href='${site.url}'>${_name(site)}</a></li>` }, '')}</ul>\n${_readme()}${_buttons()}`
+    <ul>${sites.reduce( (acc, site, id) =>
+          `${acc}<li data-type='${site.type || ''}' id='${id}'><span class='counter'>${id})</span><a href='${site.url}'>${_name(site)}</a></li>`
```
Comment (ckipp01) [inline scripts/portal.js]: Is `id='${id}'` added in here for a specific reason?

**Comment by ckipp01** [comment]
> I like the changes, but as you touched @neauoire's entry also, I would like him to check it too.

I actually changed this in my pull request as well that was just merged. So that shouldn't be an issue, but I did have a question on the added id.

**Comment by dym-sh** [comment]
i was thinking about adding numerations into some css `::before` instead of html, i'll add to the PR in a bit

**Comment by ckipp01** [comment]
Thanks for this @dym-sh!

File: scripts/portal.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ function Portal (sites) {
   function _directory (sites) {
     return `
     <ul>${sites.reduce( (acc, site, id) =>
-          `${acc}<li data-type='${site.type || ''}' id='${id}'><a href='${site.url}'>${_name(site)}</a></li>`
+          `${acc}<li ${site.type ? `data-type="${site.type}"` : ''} id='${id}'><a href='${site.url}'>${_name(site)}</a></li>`
```
Comment (dym-sh) [inline scripts/portal.js]: > data-type="${site.type}"
should i add escaping of possible double quotes in site-types? like:
``` js
`data-type="${site.type.replace(/"/g, '&quot;')}"`
```

Comment (ckipp01) [reply scripts/portal.js]: My initial opinion is no since none of the site types should have a quote in them.

**Pull request merged**